elements/pl-order-blocks/dag_checker_test.py

```diff
@@ -32,6 +32,7 @@
     ['1', '2', '3', '7'],
     ['1', '5', '6', '2', '3'],
     ['1', '6', '5', '2', '3'],
+    ['2', None, '3']
```

Comment:
Thanks for adding a test! I assume this would have failed before this PR?

Reply:
Right.